Open-Rival/include/Unit.h

```diff
@@ -118,8 +49,11 @@ namespace Rival {
 
         UnitType type;
 
+        UnitAnimation animation;
+
         int player;
 
+        /// \todo getPos(), with std::atomic<std::pair<int, int>> pos instead of int x, int y would prevent race conditions
```

Comment:
Just as an aside, I don't think need to worry about race conditions given that we only have one thread, but `getPos` may be cleaner nonetheless.